package.json

```diff
@@ -38,7 +38,7 @@
     "check-rollup-globals": "ts-node --project scripts/ scripts/check-rollup-globals.ts"
   },
   "version": "9.0.0-next.0",
-  "requiredAngularVersion": "^8.0.0 || ^9.0.0-0",
+  "requiredAngularVersion": "^9.0.0 || ^10.0.0-0",
```

Comment:
Will this match `9.0.0-next.x` releases?

Reply:
It doesn't. My thinking was that for the `next` version it doesn't matter if people get peer dependency warning. I'll update it to support pre-releases, but then we need to remember updating the range when releasing v9 stable.

Reply:
Yes, we should do that. This would be a nice thing to add to the release script (ensuring the corresponding Angular version requirement is correct)

Reply:
Have it on my list 👍 PR is updated and should be good for another review.

Reply:
Which would be:
```
`N.x.x` requires angular `^N.0.0 || (N+1).0.0-0`
`N.x.x-x` requires angular `^N.0.0-0 || (N+1).0.0-0`
```